config/locales/en.yml

```diff
@@ -39,3 +39,16 @@ en:
               blank: "Enter an email"
               taken: "This email address is already in use"
               invalid: "Enter a valid email address"
+  activemodel:
+    errors:
+      models:
+        school_cohort_form:
```

Comment:
I think we don't need to repeat `activerecord` and `errors` and `models` - specifying it once in lines 33-35 should be anough.